crates/mitex-parser/src/syntax.rs

```diff
@@ -133,6 +133,12 @@ impl From<Token> for SyntaxKind {
             Token::CommandName(CommandName::BeginEnvironment | CommandName::EndEnvironment) => {
                 SyntaxKind::TokenCommandSym
             }
+            Token::CommandName(
+                CommandName::BeginMathInline
+                | CommandName::BeginMathDisplay
+                | CommandName::EndMathInline
+                | CommandName::EndMathDisplay,
+            ) => SyntaxKind::TokenDollar,
```

Comment:
`CommandName::{Begin,End}Math{Inline/Display}` are not dollar, so convert them into a `SyntaxKind::TokenDollar` may be confusing. We may add new token kinds for them.